package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "oni-vim",
-  "version": "0.1.8",
+  "version": "0.1.9",
```

Comment:
Just a reminder, you might want to create a new tag for this release so GitHub sees it.